modules/host/networking.nix

```diff
@@ -14,17 +13,7 @@
     };
     networks."virbr0" = {
       matchConfig.Name = "virbr0";
-      networkConfig.DHCPServer = true;
-      dhcpServerConfig = {
-        ServerAddress = "192.168.100.1/24";
-        EmitRouter = true;
-        Router = "192.168.100.2";
-      };
-      addresses = [
-        {
-          addressConfig.Address = "192.168.100.1/24";
```

Comment:
I wonder if we at least for the debug variant could retain access to the VMs from host via SSH?

Reply:
I think VM access in debug is a must. Options are limited with serial debug passthroughs.